content/using/planet.md

```diff
@@ -0,0 +1,129 @@
++++
+title = "Setting up Your Planet"
+weight = 1
+description = "Getting started with a permanent Urbit identity."
+template = "page_indiced.html"
++++
+
+A planet is a permanent Urbit identity. It can never be taken from you, and its reputation will be tied to your own. If you're not yet familiar with the Urbit network, please follow the [Getting Started guide](/using/install), which will explain the basics and walk you through using a free identity. This guide explains the next steps.
```

Comment:
> It can never be taken from you, and its reputation will be tied to your own.

A comet can never be taken away from you either. I'm not sure that "its reputation will be tied to your own" is very clear to someone not already initiated.